Dockerfile

```diff
@@ -1,35 +1,57 @@
-FROM node:20.2-alpine AS base
-# Installs global dependencies as non root user for security
+FROM node:20-alpine AS base
+
+# Default Variables
+ENV PUID=1000
+ENV PGID=1000
+
+# Build time variables
+## Allow non-root usage
 ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
 ENV PATH=$PATH:/home/node/.npm-global/bin
 
-# Uses environment variables from configuration.ts
+ENV PNPM_HOME=/pnpm
+ENV PATH=$PNPM_HOME:$PATH
+
 ENV SERVER_TZ=UTC
 ENV SERVER_PORT=8080
-ENV VOLUMES_FILES="/files"
-ENV VOLUMES_IMAGES="/images"
-ENV VOLUMES_LOGS="/logs"
-ENV VOLUMES_SQLITEDB="/db"
+
+VOLUME /files /images /logs /db
 
 # Sets timezone, installs pnpm, creates and chowns the needed volumes for the node user
-RUN apk add --no-cache tzdata curl 7zip && cp /usr/share/zoneinfo/$SERVER_TZ /etc/localtime \
-    && npm i -g pnpm \
-    && mkdir /app $VOLUMES_FILES $VOLUMES_IMAGES $VOLUMES_LOGS $VOLUMES_SQLITEDB
+RUN apk add --no-cache su-exec tzdata curl 7zip \
+    && cp /usr/share/zoneinfo/$SERVER_TZ /etc/localtime \
```

Comment:
Will this work with a different Timezone variable? If not, why do we need to keep the variable?

Reply:
No, its only when you want to quickly change it at build time. We can completely remove it

Reply:
Sure, can you remove the variable from here and set the TZ using the variable in the entrypoint.sh?

Reply:
Can be now simply specified using a `TZ` variable. Verified it by typing in `date` in the container shell.

However gamevault itself outputs currently only in UTC, must be fixed in source.